prime-router/src/main/kotlin/metadata/Mappers.kt

```diff
@@ -932,6 +937,37 @@ class ZipCodeToCountyMapper : Mapper {
         )
     }
 }
+/**
+ * [ZipCodeToStateMapper] runs a lookup using zip code and returns the single associated state. Null is returned if
+ * no zip code is provided, the zip code is not found, or if multiple records are found in the lookup.
+ */
+
+class ZipCodeToStateMapper : Mapper {
+    override val name = "zipCodeToState"
+
+    override fun valueNames(element: Element, args: List<String>): List<String> {
+        return args
```

Comment:
My kotlin isn't super strong. What does this call do?

Reply:
That provides the list of fields used by the mapper. It has to be overwritten by each mapper. Some don't need to do any processing (like this one) but other's do(like the lookup mapper).